Apply new UI for the events page and the create event page

letrungkien0210 opened the pull request:
I'm handling another pages.
Please review.

Commit:
openduty notifier plugin for uptime2

letrungkien0210 committed:
Apply the new UI

letrungkien0210 committed:
Fix the check_summary id

letrungkien0210 committed:
Update the new UI for the checks page.

letrungkien0210 committed:
Rollback the fist config database connection.

letrungkien0210 committed:
Fix the check_new pages

PAStheLoD commented:
Hello,

Thanks for the pull request, it's good to see that uptime2 is not dead!

Could you expand on your changes, and the motivations behind them?

What do you mean another pages?

I'd also ask you to separate whitespace/indentation changes from the more semantic changes, but as I'm also very prone to accidentally reindent stuff, I won't :)

letrungkien0210 commented:
Hi,

I'm applying the Metronic template to uptime2.
I added css and js file of Metronic and updated ejs syntax to keep functions is correct.

Another pages that mean have some pages which don't change to new UI yet.
Please keep the pull request after I change all of pages in uptime2.

About separate whitespace/indentation changes from the more semantic changes, may be I set auto format convention on my IDE (using Visual Studio Code)

letrungkien0210 committed:
Update the new UI for create/edit event page.

letrungkien0210 committed:
Show the actived menu
